index.html

```diff
@@ -64,9 +64,11 @@ <h2><i class="fas fa-bullhorn fa-sm"></i> NUnit is a part of the .NET Foundation
                 <p>
                   The NUnit Project is a member of the <a href="https://www.dotnetfoundation.org">.NET Foundation</a>.
                   NUnit is run by the core team, <a href="https://www.alteridem.net">Rob Prouse</a>,
-                  <a href="https://www.charliepoole.org/">Charlie Poole</a>, <a href="https://Hermit.no">Terje Sandstrom</a>,
+                  <a href="https://Hermit.no">Terje Sandstrom (Lead)</a>,
                   <a href="https://github.com/jnm2">Joseph Musser</a>
-                  and <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>.
+                  <a href="https://github.com/mikkelbu">Mikkel Nylander Bundgaard</a>,
+                  <a href="https://github.com/stevenaw">Steven Weerdenburg</a>,
+                  and <a href="https://SeanKilleen.com">Sean Killeen</a>
```

Comment:
```suggestion
                  and <a href="https://SeanKilleen.com">Sean Killeen</a>.
```

Add period back. I need coffee this morning.